pkg/daemon/daemon.go

```diff
@@ -675,16 +683,16 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			ptpClockThreshold: getPTPThreshold(nodeProfile),
 			haProfile:         haProfile,
 			syncERelations:    relations,
-			profileClockType:  profileClockType,
 		}
+
 		if pProcess == ptp4lProcessName {
-			if profileClockType == "T-BC" {
+			if clockType == event.BC {
 				dprocess.trIfaceName, _ = (*nodeProfile).PtpSettings["upstreamPort"]
```

Comment:
This will break , since this is not applied for regular BC and should use check for  key/value availability  before setting